holidays/countries/united_kingdom.py

```diff
@@ -90,22 +93,29 @@ def _populate(self, year: int) -> None:
             self._add_holiday("Spring Bank Holiday", dt)
 
         # Christmas Day
-        dec_25 = self._add_christmas_day("Christmas Day")
+        self._add_observed(self._add_christmas_day("Christmas Day"), days=+2)
```

Comment:
Please ignore this one as its seems that while the celebration itself was banned until then, [it was nonetheless declared as a Bank Holiday in Scotland itself since 1871](https://books.google.co.th/books?id=smQwAQAAMAAJ&pg=PA131&redir_esc=y#v=onepage&q=holidays&f=false)
![image](https://github.com/dr-prodigy/python-holidays/assets/19505219/ff39ae85-f486-4380-896f-7671808fb54e)